content/en/software/using-medley/_index.md

```diff
@@ -93,7 +75,39 @@ These documents were converted from Medley's internal format into PDFs. Watch ou
 ### Interlisp Books
 
 - [Interlisp - The Language and Its Usage](/documentation/1986-interlisp-language-book-1.pdf)
-   Now also available in [zipped DJVU format](/documentation/1986-Interlisp-Language-Usage.ocr.djvu.zip)
+  Also available in [zipped DJVU format](/documentation/1986-Interlisp-Language-Usage.ocr.djvu.zip)
   - <a href="20211225-interlisp-book-2.pdf">Medley Interlisp: Interactive Programming Environment (derived from Interlisp-D)</a>
   - <a href="2021-interlisp-book-3.pdf">Medley Interlisp: Interactive Programming Tools (derived from Interlisp-D)</a>
 - <a href="IRM.pdf">(1993) Interlisp Reference Manual</a>
+
+# Another brief guideline for online users
```

Comment:
Should this whole section be broken into a separate page?  It seems out of place appended to the bottom of this document.

Reply:
i moved it into a separate document. There are now three 'using' subpages:
using Medley online.

I'm still having two problems: 

Problem 1: 
I want the 'short name' of the page to appear in the left column and the top line  "Install and Run > Windows > WSL" but a single header with the 'long title' of the page
 "Install and Run on Windows with WSL" underneath.

I tried shortening the names and added a # h1 header but there's an extra line with the short name.

Problem 2:
THe 404 page tries to use a relative link to pick up its style sheet, but for pages not at the top level, it doesn't change the URL to 404.html and so "not found" pages are rendered without style. THat is, if you type 
https://interlisp.org/sfsfdjsdfjsdf   you get a nice "not found" page, but if you type  https://interlisp.org/software/asdfsdfsdf  you get bad behavior.

Is this an odd artifact of gh-pages?